docs/sources/alerting/monitor/meta-monitoring/_index.md

```diff
@@ -140,14 +174,46 @@ This metric is a gauge. It has a constant value `1`, and contains a label called
 
 This metric is a counter that shows you the number of failed peer connection attempts. In most cases you will want to use the `rate` function to understand how often reconnections fail as this may be indicative of an issue or instability in your network.
 
-> These metrics are not available in Grafana Cloud as it uses a different high availability strategy than on-premise Alertmanagers.
+## Monitor your metrics
+
+To monitor your metrics, you can:
+
+1. [Optional] Create a dashboard in Grafana that uses this metric in a panel (just like you would for any other kind of metric).
+1. [Optional] Create an alert rule in Grafana that checks this metric regularly (just like you would do for any other kind of alert rule).
+1. [Optional] Use the Explore module in Grafana.
+
+## Use insights logs
+
+{{% admonition type="note" %}}
+For Grafana Cloud only.
+{{% /admonition %}}
+
+Use insights logs to help you determine which alerting and recording rules are failing to evaluate and why. These logs contain helpful information on specific alert rules that are failing, provide you with the actual error message, and help you evaluate what is going wrong.
+
+### Before you begin
+
+To view your insights logs, you must have the following:
+
+ - A Grafana Cloud account
+ - Admin or Editor user permissions for the managed Grafana Cloud instance
+
+### Procedure
+
+To explore logs pertaining to failing alerting and recording rules, complete the following steps.
+
+1. Log on to your instance and click the      **Explore** (compass) icon in the menu sidebar.
+1. Use the data sources dropdown located at the top of the page to select the data source.
+The data source name should be similar to `grafanacloud-<yourstackname>-usage-insights`.
+1. To find the logs you want to see, use the **Label filters** and **Line contains** options in the query editor.
+
+To look at a particular stack, you can filter by **instance_id** instead of **org_id**.
+
+The following is an example query that would surface insights logs:
+{org_id="<your-org-id>"} | logfmt | component = `ruler` | msg = `Evaluating rule failed`
 
-<!---
-#### cortex_prometheus_rule_group_last_evaluation_timestamp_seconds
+1. Click **Run query**.
 
-#### cortex_prometheus_rule_group_rules
+   In the **Logs** section, view specific information on which alert rule is failing and why.
 
-This metric is a counter that shows
+   You can see the rule contents ("rule"), the rule name ("name"), the name of the group it’s in ("group"), and the error message ("err").
```

Comment:
Do you think it would be helpful to have a screenshot here?
If so, here's a link to one: https://drive.google.com/file/d/1wuINfWpQk175SCnQrlAcUAiGlR42pYXa/view?usp=sharing